litert/vendors/qualcomm/core/utils/miscs.h

```diff
@@ -31,11 +34,30 @@ float Dequantize(const T val, const float scale, const int32_t zero_point) {
   return scale * (val - zero_point);
 }
 
-void ConvertDataFromInt16toUInt16(absl::Span<const std::int16_t> src,
-                                  std::vector<std::uint16_t>& dst);
-
-void ConvertDataFromUInt16toInt16(absl::Span<const std::uint16_t> src,
-                                  std::vector<std::int16_t>& dst);
+template <typename T>
+using EnableIfInt16OrUint16Ptr = std::enable_if_t<
+    std::is_same_v<T, int16_t*> || std::is_same_v<T, uint16_t*>, bool>;
+// Converts data between quantized UINT16 and INT16 formats.
+template <typename T, EnableIfInt16OrUint16Ptr<T> = true>
+void ToggleMsb(T src, size_t tensor_size) {
```

Comment:
I guess `Msb` means most significant bit, but it may not obvious for others.

Also change the name of the second parameter. 

```suggestion
void ToggleMostSignificantBit(T src, size_t size) {
```